### PR TITLE
Sonos binding: 2 new channels currenttrackuri and currenttransporturi

### PR DESCRIPTION
**Pull request by lolodomo**
This PR implements #2164.

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java [hunk 1]
```diff
@@ -560,6 +560,16 @@ public void onValueReceived(String variable, String value, String service) {
                     updateCurrentURIFormatted(value);
                     break;
                 }
+                case "AVTransportURI": {
+                    updateState(CURRENTTRANSPORTURI, (stateMap.get("AVTransportURI") != null)
```
Comment (maggu2810) [inline extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java]: There is no need to use braces around the stateMap.get call.

File: extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java [hunk 2]
```diff
@@ -560,6 +560,16 @@ public void onValueReceived(String variable, String value, String service) {
...
+                            ? new StringType(stateMap.get("AVTransportURI")) : UnDefType.UNDEF);
+                    break;
+                }
+                case "CurrentTrackURI": {
+                    updateState(CURRENTTRACKURI, (stateMap.get("CurrentTrackURI") != null)
```
Comment (maggu2810) [inline extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java]: There is no need to use braces around the stateMap.get call.

**Comment by lolodomo** [comment]
@maggu2810 :  I put braces because all other cases in the switch have braces. But I agree they are not necessary.

**Comment by maggu2810** [comment]
You are right. I just have had a look at the changes not on the existing code. So it is okay but perhaps we could refactor the code some time.

**Pull request merged**